### PR TITLE
creat a new file with homework

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: about/3 lesson/index.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=1.0>, initial-scale=1.0">
```
Comment (ghost) [inline about/3 lesson/index.html]: `meta` tags should be closed.

File: about/3 lesson/index.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                    <input required type="email" id="email" name="email"/>
+                    <br/>
+                    <label for="password">Password</label>
+                    <br/>
+                    <input required type="password" id="password" name="password"/>
+                    <br/>
+                    <p>Your programming level</p>
+                    <input type="range" vin="1" max="20" value="10"/>
+                    <p>What does HTML stand for?</p>
+                    <input type="radio" id="html" name="html" value="option1"/>
```
Comment (ghost) [inline about/3 lesson/index.html]: `id` should be unique.

File: about/3 lesson/index.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                    <label for="html">Hyperlinks and Text Markup Language</label>
+                    <br/>
+                    <input type="radio" id="html" name="html" value="option2"/>
```
Comment (ghost) [inline about/3 lesson/index.html]: `id` should be unique.

File: about/3 lesson/index.html [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+                    <label for="html">Home Tool Markup Language</label>
+                    <br/>
+                    <input type="radio" id="html" name="html" value="option3"/>
```
Comment (ghost) [inline about/3 lesson/index.html]: `id` should be unique.

File: about/3 lesson/index.html [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+                    <label for="html">Hyper Text Markup Language</label>
+                    <p>What does HTTP stand for?</p>
+                    <input type="radio" id="http" name="http" value="option1"/>
```
Comment (ghost) [inline about/3 lesson/index.html]: `id` should be unique.

File: about/3 lesson/index.html [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+                    <label for="http">Hyper Text Translate Protocol</label>
+                    <br/>
+                    <input type="radio" id="http" name="http" value="option2"/>
```
Comment (ghost) [inline about/3 lesson/index.html]: `id` should be unique.

File: about/3 lesson/index.html [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+                    <label for="http">Hyper Text Transmission Protocol</label>
+                    <br/>
+                    <input type="radio" id="http" name="http" value="option3"/>
```
Comment (ghost) [inline about/3 lesson/index.html]: `id` should be unique.

File: about/3 lesson/index.html [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+                    <label for="http">Hyper Text Transfer Protocol</label>
+                    <p>Comment</p>
+                    <textarea name="comment" maxlength="500">Comment</textarea>
+                    <p>What does HTML stand for</p>
+                    <select>
```
Comment (ghost) [inline about/3 lesson/index.html]: `select` should have attr `name`.

File: about/3 lesson/index.html (hunk 1 above)
Comment (Natali-Pchelkina) [reply about/3 lesson/index.html]: closed the tags

File: about/3 lesson/index.html (hunk 4 above)
Comment (Natali-Pchelkina) [reply about/3 lesson/index.html]: renamed the id

File: about/3 lesson/index.html (hunk 5 above)
Comment (Natali-Pchelkina) [reply about/3 lesson/index.html]: renamed the id

File: about/3 lesson/index.html (hunk 6 above)
Comment (Natali-Pchelkina) [reply about/3 lesson/index.html]: renamed the id

File: about/3 lesson/index.html (hunk 7 above)
Comment (Natali-Pchelkina) [reply about/3 lesson/index.html]: renamed the id

File: about/3 lesson/index.html (hunk 8 above)
Comment (Natali-Pchelkina) [reply about/3 lesson/index.html]: gave the name to tag <select>

File: about/3 lesson/index.html (hunk 2 above)
Comment (Natali-Pchelkina) [reply about/3 lesson/index.html]: renamed the id

File: about/3 lesson/index.html (hunk 3 above)
Comment (Natali-Pchelkina) [reply about/3 lesson/index.html]: renamed the id

File: about/3 lesson/index.html [hunk 9]
```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
```
Comment (ghost) [inline about/3 lesson/index.html]: why did you create folder `3 lesson` in `about` directory for your home task ?). You should to do that in the root `./index.html` file. Delete `lesson 3` folder, move code to `./index.html` file.

Comment (Natali-Pchelkina) [reply about/3 lesson/index.html]: moved code to ./index.html file.

**Pull request merged**